backend/tests/test_auth.py

```
@@ -27,7 +29,9 @@ def _make_app():
     app = FastAPI()
 
     @app.get("/protected")
-    async def protected(user: dict = pytest.importorskip("fastapi").Depends(get_current_user)):
+    async def protected(
+        user: dict = pytest.importorskip("fastapi").Depends(get_current_user),
```

Ruff (0.15.6)

[warning] Do not perform function call in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)